packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -81,13 +82,18 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
             }
         });
 
+        this._onForceUpdateObserver = this.props.globalState.stateManager.onForceUpdateObservable.add(() => {

```

Comment:
nit: Maybe the onForceUpdate name could be more specific? Something like onForceUpdateProperties? Just so it's super clear what part of the app is being force updated :)

Reply:
Good point! I'll fix that